homework3.py

```diff
@@ -0,0 +1,57 @@
+#Question1
+file = open('hw3_q1.txt', 'r')
+
+lines = file.readlines()
+
+lst = []
+for line in lines:
+    line_split = line.split('')
+    first_word = line_split[0]
+    second_word = line_split[1]
+    if first_word == 'JUMP':
+        lst.insert(0,second_word)
+    elif first_word == 'JION':
+        lst.append(second_word)
+print(lst)
+
```

Comment:
Total Marks 11/15 

You missed out on right output, you used JION instead of JOIN